### PR TITLE
[WIP][POC] update swiftfin tvos signin

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## Changes made

Updated the swiftfin tvos signin view

## Before

Simulator Screenshot - Apple TV 4K (3rd generation) - 2024-06-15 at 02 17 00

## After

Simulator Screenshot - Apple TV 4K (3rd generation) - 2024-06-15 at 02 08 41
## Feedback

Let me know which one is best and what you think.

### CODE REVIEW

File: Swiftfin tvOS/Views/UserSignInView/UserSignInView.swift [hunk 1]
```diff
@@ -109,104 +107,121 @@ struct UserSignInView: View {
...
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
-                    .frame(height: 100)
-                    .edgePadding()
-            }
 
-            HStack(alignment: .top) {
-                VStack(alignment: .leading) {
-                    signInSection
+                VStack {
```
Comment (LePips) [inline Swiftfin tvOS/Views/UserSignInView/UserSignInView.swift:152]: Thank you for fixing the buttons, but this needs to be in a `HStack` like before. This is a deliberate design decision for these being different sections, and they should be equally sized. Additionally, the public user section can be of any size and requiring to scroll through them isn't desirable.

**Review by LePips — CHANGES_REQUESTED** [review]

**Comment by JPKribs** [comment]
Hi @Isaiah-Hamilton,

I hope you’re doing well! I just wanted to check in to see if you’re still interested in working on this. I’ve been revisiting some tvOS features recently, and I think there’s some potential for refining this `View`. If you’re still keen on handling it, I completely respect that and don’t want to overstep at all.

However, if you’re a bit busy or prefer someone else to take it on, I’d be happy to help out or even pick it up from here! This is the latest version I was working on based on @LePips feedback:

Simulator Screenshot - Apple TV 4K (3rd generation) - 2024-12-12 at 13 44 59

I'd be happy to share this code as well!

Let me know your thoughts! 😊

**Comment by Isaiah-Hamilton** [comment]
@JPKribs I'm a little to busy at the moment to work on this, but I would be happy to pass this over to someone else. Though I would still like to be kept in the loop/have to some input on somethings. So far what you have is great, and think we should go based on what you have currently.

**Comment by JPKribs** [comment]
> @JPKribs I'm a little to busy at the moment to work on this, but I would be happy to pass this over to someone else. Though I would still like to be kept in the loop/have to some input on somethings. So far what you have is great, and think we should go based on what you have currently.

Sounds good! Thanks for getting the ball rolling. I have my branch of this here: https://github.com/jellyfin/Swiftfin/pull/1365

Let me know if you have any feedback! I found that the `ConnectToServerView` and `UserSignInView` were both build on the same layout so my primary change from here is to reuse the same layout for both.

**Comment by JPKribs** [comment]
This work has been merged in via https://github.com/jellyfin/Swiftfin/pull/1365, so I am going to close this out.

Thank you again for getting the ball rolling with this SignIn cleanup, @Isaiah-Hamilton!

**Pull request closed without merging**